packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/template.tsx

```diff
@@ -51,7 +51,11 @@ const LinkHereButton = () => (
   </div>
 )
 
-const DebitCard = ({ modalActions }: Props) => {
+const DebitCard = ({ debitCardActions, modalActions }: Props) => {
+  useEffect(() => {
+    // Get available products that user can create
```

Comment:
can't this be on the `mapStateToProps`?

Reply:
I prefer to leave this here, for now, this might change in the following PRs and if I add to mapStateToProps I would also have to add to pass it through the props to this component. Adding a lot of code lines unnecessary

Reply:
Do you see this pattern somewhere else?

Reply:
Im sorry I don’t understand what you are trying to say

Reply:
@pedroapfilho do you mean add `debitCardActions.getProducts()` to mapDispatchToProps? mapStateToProps is only really for selectors, not actions.